### ISSUE TITLE
Backup using GUI doesn't work in Ubuntu

### ISSUE THREAD

**Issue by hotice** [movitto/snap#1]
I've tried to use Snap in Ubuntu 11.10 but it doesn't work: when trying to create a backup (repos, packages and services) using the GUI, it freezes and it creates a backup archive of 19.1kb (it's always this size). Unfortunately, no matter what I select under "log level", I only get "Creating snapshot" so I don't have the any error output to show you...

I've tried both the .deb (which is pretty old, maybe you can update it?) and the latest code via GitHub.

**Comment by movitto** [comment]
Hey thanks for the bug report. As suggested, I just updated snap.deb to the latest release 0.6 (make sure to 'apt-get update' if your using my apt repo)

A couple of questions. First are you sure you are running gsnap as an admin user (eg via sudo)? If you run gsnap from the command line, do you see any output there? 

What about if you run snaptool itself. You can accomplish this by opening up a terminal and running 'sudo snaptool -l verbose --backup --snapfile /tmp/snapfile.tgz --repos --packages --services'

Again thank you greatly for the bug report.

**Comment by hotice** [comment]
Yes, I've used it with gksu and there was no output on the command line either. Running snaptool works just fine. Only the gui seems to have issues.

**Comment by movitto** [comment]
Can you elaborate on the steps you took to reproduce this issue. I was able to use gsnap successfully on Ubuntu 11.10. Here is what I did:
- setup a new vm w/ Ubuntu Oneiric
- ran a post-install upgrade
  $ sudo apt-get upgrade
- setup the apt.morsi.org repo by adding the following to /etc/apt/sources:
  deb http://apt.morsi.org/ubuntu oneiric main
  deb-src http://apt.morsi.org/ubuntu oneiric main
- installed snap by running:
  $ sudo apt-get update
  $ sudo apt-get install snap
- Ran a snapshot via
  $ gksu gsnap
- Clicked 'backup', selected 'repos', 'packages', 'services', selected 'debug' for output level clicked 'start'
- It took a little while (it did sit on creating snapshot for a bit) but the output window was populated w/ the output and the snapshot was created.

What kind of system are you running this on, snap doesn't do too much processor or network intensive but copying files could take some time (perhaps some large database files or similar?) How long did snaptool take to run? Or perhaps you have additional services configuration causing an issue which I'm not running into. Do you have any custom iptables rules, or custom config to httpd, postgres, or mysql that could be causing issues? Any way you could share the packages you have installed locally so I can see if something there is choking things up? (dpkg-query -W *)

Appreciate it.
